config/settings.yml

```diff
@@ -50,6 +50,10 @@ environment:
   label: "Beta"
   selector_name: "beta"
 current_cycle: 2020
+# `financial_support_placeholder_cycle` the cycle year value should be 
+# omitted if placeholder is not required otherwise the it should be the
+# new cycle.
```

Comment:
Just wondering if its usual to have comments like this in the code?

Reply:
I think it's ok where the code isn't self documenting and there's a potential for confusion or a lack of clarity. Obviously, we should write self documenting code that isn't confusing but we all have off days 🤣

Reply:
it really confusing... and i never have a non-off day

it that period where it goes from 2 (current/next) cycles and back to 1 (new/current) cycle